repo/installer.go

```diff
@@ -248,6 +248,16 @@ func (i *Installer) Export(conf *cfg.Config) error {
 	if err != nil {
 		return err
 	}
+	// defer func() {
+	// 	err = os.RemoveAll(tempDir)
```

Comment:
oh, i may have had this problem before, if the issue was `os.RemoveAll()` failing on windows? if so, it should be fixed in go1.7, but in the meantime, [this](https://github.com/sdboyer/gps/blob/master/remove_go16.go) will fix it